src/components/Incidents/AlertsChart/AlertsChart.jsx

```diff
@@ -1,140 +1,167 @@
 import * as React from 'react';
 
-import { Chart, ChartAxis, ChartBar, ChartGroup, ChartLabel, ChartThemeColor } from '@patternfly/react-charts';
+import {
+  Chart,
+  ChartAxis,
+  ChartBar,
+  ChartGroup,
+  ChartLabel,
+  ChartThemeColor,
+} from '@patternfly/react-charts';
 import { VictoryZoomContainer } from 'victory-zoom-container';
 import { Card, CardTitle } from '@patternfly/react-core';
 
-const days = ['Jul 25', 'Jul 26', 'Jul 27', 'Jul 28', 'Jul 29', 'Jul 30', 'Jul 31', 'Aug 1', 'Aug 2', 'Aug 3', 'Aug 4', 'Aug 5', 'Aug 6', 'Aug 7', 'Today',]
+const days = [
+  'Jul 25',
+  'Jul 26',
+  'Jul 27',
+  'Jul 28',
+  'Jul 29',
+  'Jul 30',
+  'Jul 31',
+  'Aug 1',
+  'Aug 2',
+  'Aug 3',
+  'Aug 4',
+  'Aug 5',
+  'Aug 6',
+  'Aug 7',
+  'Today',
+];
 const AlertsChart = () => {
   return (
-    <Card>
+    <Card
+      style={{
+        border: '6px solid #F3F3F3',
```

Comment:
This should probably come from patternfly CSS vars as this will need to change if user is in dark mode

Reply:
There is no mocks for darkmode. so what should I do with it?

Reply:
Patternfly components support dark mode so we should use the CSS variables which already adjust to it instead of using hardcoded colors. In this specific case we might not need to change the color, just the border width